examples/07-routing/src/app.gleam

```diff
@@ -158,16 +141,41 @@ fn render_welcome(model: Model, slug) -> Element(a) {
     |> list.find(fn(guest: Guest) { guest.slug == slug })
 
   let title = case guest {
-    Ok(guest) -> to_title("Hello, " <> guest.name <> "! 🎉")
-    _ -> to_title("Sorry ... didn't quite catch that.")
+    Ok(guest) -> render_title("Hello, " <> guest.name <> "! 🎉")
+    _ -> render_title("Sorry ... didn't quite catch that.")
+  }
+
+  render_body([title])
+}
+
+fn render_nav(model: Model) -> Element(a) {
+  let item_styles = [#("margin", "1rem"), #("text-decoration", "underline")]
+
+  let render_nav_item = fn(path, text) {
+    html.a([attribute.href("/" <> path), attribute.style(item_styles)], [
+      element.text(text),
+    ])
   }
 
-  ui.centre([attribute.style([#("margin-top", "10vh")])], title)
+  let guest_nav_items =
+    model.guests
+    |> list.map(fn(guest: Guest) {
+      render_nav_item("welcome/" <> guest.slug, guest.name)
+    })
+
+  html.nav([], [render_nav_item("", "Home"), ..guest_nav_items])
+}
+
+fn render_body(children) {
+  ui.centre(
+    [attribute.style([#("margin-top", "10vh")])],
+    ui.stack([], children),
+  )
 }
 
-fn to_title(text) {
+fn render_title(text) {
   html.h1(
-    [attribute.style([#("font-size", "36px"), #("font-weight", "bold")])],
+    [attribute.style([#("font-size", "2.25rem"), #("font-weight", "bold")])],
```

Comment:
Let's use `cn.text_lg()` or `cn.text_xl()`

Reply:
Updated this and the header image as well